sass/locales/_fr.scss

```
@@ -69,6 +69,7 @@ $messages: (
   "error:role-scrollbar": "Attributs manquants pour [role=scrollbar]",
   "error:nested-interactive": "Éléments interactifs imbriqués",
   "error:show-lang": "attr(lang)",
+  "error:attribute-whitespace": "Attribute can't contain whitespace",
```

Here's a french one :)

```suggestion
  "error:attribute-whitespace": "Attribut ne devant pas contenir d’espace",
```